inductiva/resources/machine_groups.py

```diff
@@ -72,10 +75,10 @@ def list():
     that belong to the user. It outputs all the information relative to
     each machine group as folllows:
 
-    INFO:absl:Active machine groups:
-                                        Name         VM Type   Elastic  # machines    Disk Size in GB       Spot         Started at
-    api-1b1f724c-5cfe-4d87-8439-9689aa139723   c2-standard-4      True   1                 40      False   13 Sep, 07:38:50
-    api-8e6bf7d8-4888-4de9-bda5-268484b46e6f   c2-standard-4     False   1                 40      False   13 Sep, 07:37:49
+    Active machine groups:
+                                        Name         VM Type    Elastic   # machines    Disk Size in GB       Spot   Started at (UTC)
+    api-6359c03d-c4f9-479f-8b11-ba1f8f55a58c   e2-standard-4      False            3                 40      False   10 Oct, 13:40:50
```

Comment:
Why just 3?

Reply:
I just commented, it is because non-elastic machines need more work done in the background to get the initially requested number of machines.